portal/tasks.py

```diff
@@ -182,41 +212,81 @@ def update_patient_loop(update_cache=True, queue_messages=True):
     valid_patients = User.query.join(UserRoles).filter(and_(
         User.id == UserRoles.user_id,
         User.deleted_id.is_(None),
-        UserRoles.role_id == patient_role_id))
+        UserRoles.role_id == patient_role_id)).with_entities(User.id)
+
+    patients = valid_patients.all()
+    j = 0
+    batchsize = 16
```

Comment:
Can you make this configurable? much easier to test on stg* than editing code. would be willing to remove configurability later once we've settled on a good default

Reply:
sure.